lib/cssom.js

```diff
@@ -35,7 +35,26 @@ declare class CSSStyleSheet extends StyleSheet {
   cssRules: CSSRuleList;
   ownerRule: ?CSSRule;
   deleteRule(index: number): void;
-  insertRule(rule: string, index: number): void;
+  insertRule(rule: string, index: number): number;
+}
+
+declare class CSSGroupingRule extends CSSRule {
+  +cssRules: CSSRuleList;
+  deleteRule(index: number): void;
+  insertRule(rule: string, index: number): number;
+}
+
+declare class CSSConditionRule extends CSSGroupingRule {
+  conditionText: string;
+}
+
+declare class CSSMediaRule extends CSSConditionRule {
+  media: MediaList;
```

Comment:
This should be read-only, right? https://www.w3.org/TR/2016/WD-cssom-1-20160317/#the-cssmediarule-interface

Reply:
yep, I just went through all added interfaces and some previously existing once and updated all readonly props